internal/component/kafka/consumer.go

```diff
@@ -44,7 +44,7 @@ func (consumer *consumer) ConsumeClaim(session sarama.ConsumerGroupSession, clai
 			}, func() {
 				consumer.k.logger.Infof("Successfully processed Kafka message after it previously failed: %s/%d/%d [key=%s]", message.Topic, message.Partition, message.Offset, asBase64String(message.Key))
 			}); err != nil {
-				return err
+				consumer.k.logger.Errorf("Error processing Kafka message: %s/%d/%d [key=%s]. Error: %v.", message.Topic, message.Partition, message.Offset, asBase64String(message.Key), err)
```

Comment:
To differentiate this error message, can you please log the number of failed attempts too? For example:

`Failed processing Kafka message after %d attempts: %s/%d/%d [key=%d]. Error: %v.`

Reply:
> To differentiate this error message, can you please log the number of failed attempts too? For example:
> 
> `Failed processing Kafka message after %d attempts: %s/%d/%d [key=%d]. Error: %v.`

The `retry.NotifyRecover` seems can not get attempts times when failed.

Reply:
How about this `Failed processing Kafka message after it previously failed: %s/%d/%d [key=%d]. Error: %v.` ?

Reply:
Sounds good, but if I can wordsmith a bit: "Too many failed attempts at processing Kafka message." (+ the rest)

Reply:
done